air/src/options.rs

```diff
@@ -92,6 +92,7 @@ pub struct ProofOptions {
     fri_folding_factor: u8,
     fri_remainder_max_degree: u8,
     partition_options: PartitionOptions,
+    batching_deep: BatchingType,
```

Comment:
nit: I would move this above `partition_options`.

Also, let's add a paragraph to the doc comments above to explain the meaning of this parameter (maybe re-use the text I suggested in the above comment).

Reply:
Done